src/domains/canoneunico/99_main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
```

Comment:
please add the others providers defined into terraform.lock.hcl to avoid prolems in future

Reply:
done